lib/mongo.ex

```diff
@@ -185,6 +185,21 @@ defmodule Mongo do
   end
 
   @doc """
+  Selects a single document in a collection and return the document or null
+  """
+  @spec findOne(Pool.t, collection, BSON.document, Keyword.t) :: BSON.document
+  def findOne(pool, coll, filter, opts \\ []) do
```

Comment:
The mongo driver specifications tells us to follow the language conventions, which is to use snake_case.